03/app.js

```diff
@@ -8,7 +8,33 @@ const buttonSettings = {
     css: {
         border: '1px solid #336699',
         padding: '5px 20px',
-        color: '#444'
+        color: '#444',
+        cursor: 'pointer'
     },
     text: 'Click me!',
+}
+
+const parent = document.querySelector('.parent-for-button')
+
+if(parent) {
+
+    const btn = document.createElement('button');
+
+    for(const key in buttonSettings) {
+
+        if( key === 'attr' )  {
+            for(const key in buttonSettings.attr) {
+                btn.setAttribute(key , buttonSettings.attr[key])
+            }
+        }
+        else if( key === 'css' ) {
+            for(const key in buttonSettings.css) {
+                btn.style.setProperty( key , buttonSettings.css[key])
```

Comment:
Można by było również `btn.style[key] =  buttonSettings.css[key]`

Reply:
Jestem świadomy, testowałem nowo poznane metody i postanowiłem zrobić to w ten sposób. Czy jest jakaś przewaga jednej opcji nad drugą?.